pom.xml

```diff
@@ -244,7 +242,7 @@
         <jts.spatial4j.version>1.15.0</jts.spatial4j.version>
         <jts.version>1.14.0</jts.version>
         <jvnet-ogc.version>2.6.1</jvnet-ogc.version>
-        <karaf.version>4.2.1</karaf.version>
+        <karaf.version>4.2.2</karaf.version>
```

Comment:
✏️[[KARAF-6046] Upgrade to Felix Utils 1.11.2](https://github.com/apache/karaf/commit/5fc03863b91db44dc5afbc341a2477c1f362bca9)
https://github.com/codice/ddf/blob/7f61ab9478cc54fe1ddf6a1c701f98c84965ce0b/platform/io/platform-io-impl/pom.xml#L40-L44
```xml
<dependency>
    <groupId>org.apache.felix</groupId>
    <artifactId>org.apache.felix.utils</artifactId>
</dependency>
```